### ISSUE TITLE
PAH fails to follow the first Tsar selection method

### ISSUE THREAD

**Issue by domcleal** [grifferz/pah-irc#104]
It could go further to select the first Tsar, by requesting data from players to make the correct decision (as stated in the rulebook).

https://github.com/grifferz/pah-irc/blob/v0.4/lib/PAH.pm#L1491

**Comment by grifferz** [comment]
I don't think having it ask everyone when they last pooed is going to work. It will slow down the start of the game while it waits for everyone to respond. Could time limit it I suppose, e.g.:
- `<AgainstHumanity>` The game is going to start in one minute. When did you last poo?

Parse times in a variety of formats, assume everyone that doesn't answer pooed at the same time as the one who pooed longest ago.

Even so, I have a feeling that having to wait one minute is going to wind some people up.

I wonder if there is any other selection method that could be used automatically.

IRC idle time? It'd always pick the last person to join the game then though.

**Comment by domcleal** [comment]
A time limit seems reasonable, though I'm not sure that it needs to be that short - it could be equal to the length of the turn.

Doesn't idle time also only show up after a certain amount of time or similar?  It isn't always visible in a /whois for people who've been actively recently.

I think an automatic method might detract from what's meant to be a slightly fun way of choosing the first Tsar.  Choosing automatically might as well be random, or the person starting the game as it is now.

**Comment by turmoni** [comment]
> Doesn't idle time also only show up after a certain amount of time or similar? It isn't always visible in a /whois for people who've been actively recently.

It only shows up with a normal whois if they're on the same server as you; most clients will allow you to "/wii" someone, or alternatively you can duplicate their nick (/whois nick nick) to do a remote whois.

**Comment by domcleal** [comment]
Interesting, thanks!

**Comment by grifferz** [comment]
I agree with @domcleal about an automatic method detracting from the fun. Unless it was a fun automatic method. But I can't think of one off the top of my head.

I'm not sure that a channel that is new to this (version of the) game will take well to being asked when they pooed and having it wait up to 24 hours for everyone to answer!

It will also be hard to make it a per-channel setting because the game only starts once in each channel, so there is no way to change the setting beforehand.
